services/els-verify/src/main/java/app/coronawarn/datadonation/services/els/otp/ElsOtpControllerExceptionHandler.java

```diff
@@ -13,17 +13,17 @@
 @ControllerAdvice
 public class ElsOtpControllerExceptionHandler extends ResponseEntityExceptionHandler {
 
-  private static final Logger logger = LoggerFactory.getLogger(ElsOtpControllerExceptionHandler.class);
+  private static final Logger ELTS_LOGGER = LoggerFactory.getLogger(ElsOtpControllerExceptionHandler.class);
```

Comment:
Maybe for code consistency we can keep this logger with the name LOGGER as seen above.

Reply:
that's one of the blocker / critical raised by sonar. the naming of the Loggers - one should not have the same variable naming as in the upper class.

Reply:
what about making the logger protected in parent class and reusable in subclasses?

Reply:
then the logs will be as coming from the parent class, which might not be ideal